jasmine-standalone-2.5.2/build/bundle.js

```diff
@@ -204,6 +204,17 @@ describe('InvertedIndex', () => {
       expect(invIndex.validateFile(valid2, 'valid2.json')).toBeTruthy();
     });
   });
+
+  describe('searchIndex', () => {
+    it('Should be able to get indices of words from already indexed files', () => {
+      invIndex.createIndex(valid3, 'valid3.json');
+      invIndex.createIndex(valid4, 'valid4.json');
+      expect(invIndex.searchIndex('could', 'valid4.json')).toEqual([ 0, 1 ]);
+      expect(invIndex.searchIndex('testing', 'valid4.json')).toEqual([ 0 ]);
+      expect(invIndex.searchIndex('pass', 'valid4.json')).toEqual([ 1 ]);
+      expect(invIndex.searchIndex('the', 'valid3.json')).toEqual([ 0, 1 ]);
+    });
+  });
 });
 
 },{"../../invalid1.json":1,"../../invalid2.json":2,"../../invalid3.json":3,"../../invalid4.json":4,"../../invalid5.json":5,"../../invalid6.json":6,"../../invalid7.json":7,"../../valid1.json":9,"../../valid2.json":10,"../../valid3.json":11,"../../valid4.json":12}],9:[function(require,module,exports){
```

Comment:
care to explain why this is here?... if it is a bundled file then I think its meant to be in gitignore